natlib/svcd.c

```diff
@@ -1,9 +1,13 @@
-//This file is included into native.c
+#include "libstormarray.h"
```

Comment:
This will be included in native.c by https://github.com/UCB-IoET/ioet_contrib/pull/30

Group: Leonard Truong, Sam Kumar, Michael Chen